kubernetes-templates/weave-cloud/config.yml

```diff
@@ -0,0 +1,6 @@
+name: "Weave Cloud Suite"
+description: |
+  All you need to implement Continuous Delivery with hosted Prometheus monitoring and visual distributed system analysis/debug tools
```

Comment:
"Weave Cloud is a hosted add-on to Kubernetes which provides Continuous Delivery, Prometheus Monitoring and a visual dashboard for troubleshooting & debugging microservices. This package contains the agents which connect your cluster to Weave Cloud. You will need a service token which you can get from https://cloud.weave.works/"